pkg/descheduler/strategies/types.go

```diff
@@ -39,16 +43,60 @@ type StrategyFunction func(
 
 // StrategyController is a controller responsible for running an individual strategy, used with informed strategies
 type StrategyController struct {
-	ctx    context.Context
-	client clientset.Interface
-	queue  workqueue.RateLimitingInterface
-	f      StrategyFunction
+	ctx                   context.Context
+	client                clientset.Interface
+	queue                 workqueue.RateLimitingInterface
+	sharedInformerFactory informers.SharedInformerFactory
+	stopChannel           chan struct{}
+
+	f            StrategyFunction
```

Comment:
s/f/func to avoid one letter variable (at least two letters to make searching for it easy please :))